qir/qat/RemoveDisallowedAttributesPass/RemoveDisallowedAttributesPass.hpp

```diff
@@ -25,10 +25,20 @@ namespace quantum
 
         llvm::PreservedAnalyses run(llvm::Module& module, llvm::ModuleAnalysisManager& /*mam*/)
         {
+            std::string const LLVM_FUNC_NAME = "@llvm.";
             for (auto& fnc : module)
             {
                 std::unordered_set<String> to_keep;
+                auto                       name = static_cast<std::string>(fnc.getName());
 
+                // Skipping any LLVM function
```

Comment:
Why are we leaving in commented out code?

Reply:
This is a mistake. This PR should not touch anything in `qat`.